### PR TITLE
Refactor: get rid of com.pubnub.kmp.PubNub

### PR DESCRIPTION
**Pull request by wkal-pubnub**

### CODE REVIEW

File: pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt [hunk 1]
```diff
@@ -72,13 +72,13 @@ import com.pubnub.api.v2.entities.ChannelGroup
...
 import com.pubnub.api.v2.subscriptions.Subscription
+import com.pubnub.api.v2.subscriptions.SubscriptionOptions
 import com.pubnub.api.v2.subscriptions.SubscriptionSet
 import com.pubnub.internal.BasePubNubImpl
 import com.pubnub.kmp.CustomObject
 import java.io.InputStream
 
-interface PubNub :
-    com.pubnub.kmp.PubNub,
+actual interface PubNub :
```
Comment (marcin-cebo) [inline pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt:81]: Is there a need to have `actual` interface PubNub when there is `expect interface PubNub` ?

File: pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/v2/PNConfigurationImpl.kt [hunk 2]
```diff
@@ -63,142 +63,142 @@ class PNConfigurationImpl(
         BasePNConfigurationImpl.Builder(defaultConfiguration),
         PNConfiguration.Builder,
         PNConfigurationOverride.Builder {
-            private val log = LoggerFactory.getLogger(this::class.simpleName)
+        private val log = LoggerFactory.getLogger(this::class.simpleName)
```
Comment (marcin-cebo) [inline pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/v2/PNConfigurationImpl.kt:66]: Was previous formatting improper?

File: pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt (hunk 1 above)
Comment (wkal-pubnub) [reply pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt:81]: yes, there must be an actual for each target

File: pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/v2/PNConfigurationImpl.kt (hunk 2 above)
Comment (wkal-pubnub) [reply pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/v2/PNConfigurationImpl.kt:66]: yeah the auto formatter changed this for some reason

File: pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt (hunk 1 above)
Comment (marcin-cebo) [reply pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/PubNub.kt:81]: In those examples there one interface(there is no actual version of expect interface)
https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-connect-to-apis.html#expected-and-actual-functions

https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-connect-to-apis.html#different-entry-points

File: pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/v2/PNConfigurationImpl.kt (hunk 2 above)
Comment (marcin-cebo) [reply pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/v2/PNConfigurationImpl.kt:66]: Looks bit strange to me.

Comment (marcin-cebo) [reply pubnub-kotlin/pubnub-kotlin-impl/src/main/kotlin/com/pubnub/internal/v2/PNConfigurationImpl.kt:66]: Now I get it.

**Pull request merged**